### PR TITLE
If this doesn't work I may cry.

### PR DESCRIPTION
**Pull request by kemp3673**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,19 +16,21 @@
     "@fontsource/roboto": "^4.5.8",
     "@mui/material": "^5.10.0",
     "@mui/styled-engine-sc": "^5.10.0",
+    "angular": "^1.8.3",
```
Comment (sbirvin1s) [inline package.json]: is angular needed as a dependency?

**Review by sbirvin1s — APPROVED** [review]
👍 Looks good! Had one question about using `Angular` as a dependency

**Review by jeankayy — APPROVED** [review]
looks great! super cool

File: package.json (hunk 1 above)
Comment (kemp3673) [reply package.json]: In my travels of trying to get it working was trying an angular approach. Will remove from dependencies. Good eye.

File: package.json [hunk 2]
```diff
@@ -16,19 +16,21 @@
...
-    "cors": "^2.8.5",
     "firebase": "^9.9.2",
+    "multer": "^1.4.5-lts.1",
     "nodemon": "^2.0.19",
     "path": "^0.12.7",
     "pg": "^8.7.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-firebase-hooks": "^5.0.3",
+    "react-google-login": "^5.2.2",
```
Comment (sbirvin1s) [inline package.json]: Follow up, noticed `react-google-login` is causing an pnm install conflict because it is looking for React 16 or 17

**Review by sbirvin1s — COMMENTED** [review]
Identified another `package.json` conflict that was previously missed

**Pull request merged**